plugins/keycloak-backend/src/lib/read.ts

```diff
@@ -165,24 +190,61 @@ export const readKeycloakRealm = async (
     options?.userQuerySize,
   );
 
-  const rawKGroups = await getEntities(
+  const topLevelKGroups = (await getEntities(
     client.groups,
     config,
     options?.groupQuerySize,
+  )) as GroupRepresentationWithParent[];
+
+  const serverInfo = await client.serverInfo.find();
```

Comment:
This is a REST API call to retrieve the server info.  Can we have some error handling if the server gives a bad response?